src/cs50/__init__.py

```diff
@@ -1,6 +1,28 @@
 import os
 import sys
 
+
+class CustomImporter(object):
```

Comment:
Unless I'm missing something, all this does is alias `cs50.sql` to `cs50.SQL`. Flask indirectly import `cs50.sql` anyway, so does this not help in lazy loading.